_episodes/01-quickstart.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Quickstart guide"
+teaching: 1
+exercises: 10
+questions:
+
+- "What is the purpose of the quickstart guide?"
+- "How do I activate and test the ESMValTool environment?"
+- "How do I configure ESMValTool?"
+- "How do I run a recipe?"
+
+objectives:
+
+- "Understand the purpose of the quickstart guide"
+- "Activate and test the ESMValTool environment"
+- "Configure ESMValTool"
+- "Run a recipe"
+
+keypoints:
+
+- "The purpose of the quickstart guide is to enable a user of ESMValTool
+  to run ESMValTool as quickly as possible"
+- "Use the `module load` command to activate the ESMValTool environment 
+  (see the installation guide for more details) and use `esmvaltool -- --help` to 
+  test the ESMValTool environment"
```

Comment:
The modules documentation says the purpose of the module load command is to update the environment. See https://modules.readthedocs.io/en/latest/module.html. Since the documentation says environment, I think we should also use it to avoid confusion. If activate sounds too conda specific I could change it to "load the ESMValTool environment" as a compromise?

Reply:
9f31e4f changed it.